container/endpoint.go

```diff
@@ -211,7 +211,7 @@ func buildExportedEndpoints(conn coordclient.Connection, tenantID string, state
 			exp.endpointName = defep.Name
 
 			var err error
-			ep, err := buildApplicationEndpoint(state, &defep)
+			ep, err := dao.BuildApplicationEndpoint(state, &defep)
```

Comment:
OK - I'll move ApplicationEndpoint to domain.  In the next PR, I plan on introducing Request and Result objects to the RPC for GetServiceEndpoints. As a part of that change, I plan on moving the interface out of dao completely so that the RPC layer for GetServiceEndpoints calls into the facade, bypassing the dao.